### ISSUE TITLE
after installing hero a lot of errors comes up!

### ISSUE THREAD

**Issue by nadia-am** [HeroTransitions/Hero#548]
![screen shot 2018-10-09 at 10 30 35 pm](https://user-images.githubusercontent.com/11175468/46692033-109d7680-cc13-11e8-9bc0-42886f61fb26.png)

**Comment by tangjinzhao** [comment]
me too

**Comment by boardmain** [comment]
idem

**Comment by markhmwong** [comment]
I've installed the Hero via cocoapods yesterday and if you read the errors, they're mostly refactoring issues. Go through each one and it should work. I've got mine to run without errors.

**Comment by nadia-am** [comment]
yes its refactoring , but i didn't want to change the library, is there any way to solve it?

**Comment by nadia-am** [comment]
after refactoring new errors comes up!
![3](https://user-images.githubusercontent.com/11175468/46917619-0c3bd980-cfd6-11e8-8a3b-4d86b90d41ce.png)
